stepper.md

```diff
@@ -35,30 +36,168 @@ motor drivers). Additional delay values can be added in the future.*
                                         XXXX001 = step + direction driver
                                         XXXX010 = two wire
                                         XXXX100 = four wire)
-5  steps-per-revolution LSB
-6  steps-per-revolution MSB
-7  motorPin1 or directionPin number  (0-127)
-8  motorPin2 or stepPin number       (0-127)
-9  [only when interface = 0x04] motorPin3 (0-127)
-10 [only when interface = 0x04] motorPin4 (0-127)
-11 END_SYSEX                         (0xF7)
+ * 5  steps-per-revolution LSB
+ * 6  steps-per-revolution MSB
+ * 7  motorPin1 or directionPin number  (0-127)
+ * 8  motorPin2 or stepPin number       (0-127)
+ * 9  [only when interface = 0x04] motorPin3 (0-127)
+ * 10 [only when interface = 0x04] motorPin4 (0-127)
+ * 11 END_SYSEX                         (0xF7)
+ * -----------------------------------------------------
+ */
+```
+
+###Stepper step
+```c
+ /* -----------------------------------------------------
+ * 0  START_SYSEX          (0xF0)
+ * 1  Stepper Command      (0x72)
+ * 2  config command       (0x01)
+ * 3  device number        (0-5)
+ * 4  direction            (0-1) (0x00 = CW, 0x01 = CCW)
+ * 5  num steps byte1 LSB
+ * 6  num steps byte2
+ * 7  num steps byte3 MSB  (21 bits (2,097,151 steps max))
+ * 8  [optional] speed LSB (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
+ * 9  [optional] speed MSB
+ * 10 [optional] accel LSB (acceleration in 0.01*rad/sec^2 (1000 = 10.0 rad/sec^2))
+ * 11 [optional] accel MSB
+ * 12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
+ * 13 [optional] decel MSB
+ * 14 END_SYSEX            (0xF7)
+ * -----------------------------------------------------
+ */
+```
+
+###Report Steppers's position
+
+Query
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command            (0x72)
+ * 2 Stepper Get Position       (0x02)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
+```
+ 
+Response 
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command      		(0x72)
+ * 2 Stepper Get Position       (0x02)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 current position, bits 0-6
+ * 5 current position, bits 7-13
+ * 6 current position, bits 14-20
+ * 7 current position, bits 21-27
+ * 8 the signed value			(negative 0x00 or positive 0x01)
+ * 9 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
 ```
 
-Stepper step
+###Report Steppers's Distance from target
```

Comment:
Again, this may be inaccurate once the response is received depending on the current stepping rate.